tests/Common/Api/ProductMediaFile/CreateProductMediaFileApiIntegration.php

```diff
@@ -48,6 +48,79 @@ public function testCreateSuccessful()
         $this->assertSameContent($expectedMediaFile, $mediaFiles[4]);
     }
 
+    public function testCreateForProductModelSuccessful()
+    {
+        $client = $this->createClient();
+        $api = $client->getProductMediaFileApi();
+        $baseUri = $this->getConfiguration()['pim']['base_uri'];
+        $mediaFile = realpath(__DIR__ . '/../../../fixtures/akeneo.png');
+
+        $response = $client->getAttributeApi()->create('product_model_media', [
+            'type'                   => 'pim_catalog_image',
+            'group'                  => 'media',
+            'unique'                 => false,
+            'useable_as_grid_filter' => false,
+            'max_characters'         => null,
+            'validation_rule'        => null,
+            'validation_regexp'      => null,
+            'localizable'            => true,
+            'scopable'               => false,
+            'labels'                 => [
+                'en_US' => 'product model media',
+            ],
+        ]);
+
+        $response = $client->getFamilyApi()->upsert('boots', [
+            'attributes' => [
+                'product_model_media',
+                'color',
+                'description',
+                'manufacturer',
+                'name',
+                'price',
+                'side_view',
+                'size',
+                'sku',
+                'heel_color',
+            ],
+        ]);
+
+        $response = $api->create($mediaFile, [
+            'code'       => 'rain_boots',
+            'attribute'  => 'product_model_media',
+            'scope'      => null,
+            'locale'     => 'en_US',
+            'type'       => 'product_model',
+        ]);
+
+        $this->assertSame(MediaSanitizer::MEDIA_ATTRIBUTE_DATA_COMPARISON, MediaSanitizer::sanitize($response));
+
+        $mediaFiles = $api->listPerPage(10)->getItems();
+
+        $this->assertCount(5, $mediaFiles);
```

Comment:
it will break the test if we add a media in the fixtures, maybe just iterate the items to check that it exists.
(by copy + rename original filename just for the test for example)